httpserver/http_connection.cpp

```diff
@@ -748,17 +747,18 @@ void connection_t::process_store(const json& params) {
 
     // Do not store message if the PoW provided is invalid
     std::string messageHash;
+    const pow_difficulty_t curr_difficulty = service_node_.get_pow_difficulty();
 
-    const bool validPoW =
+    const bool valid_pow =
         checkPoW(nonce, timestamp, ttl, pubKey, data, messageHash,
-                 service_node_.get_pow_difficulty());
+                  std::vector<pow_difficulty_t>{curr_difficulty});
```

Comment:
Having to create a vector on every store request seems inefficient. Could we have an overload for this that takes a single value? (Or we could have a separate function to get the minimal required pow target for the message, and then call another method to actually check the message against this target.)

Reply:
Updated